src/Gui/Dialogs/DlgAddPropertyVarSet.h

```diff
@@ -82,67 +90,62 @@ public Q_SLOTS:
     void valueChanged();
 
 private:
+    enum class TransactionOption : bool {
+        COMMIT = false,
+        ABORT = true
```

Comment:
```suggestion
        Commit = false,
        Abort = true
```

Enums should use PascalCase I think?

Reply:
Yes, I agree.